### DISCUSSION TITLE
Welcome to WooCommerce Local Pickup Time Discussions!

### DISCUSSION THREAD

**Discussion by timnolte** [mattbanks/woocommerce-local-pickup-time#107] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by An0dmin** [comment]
Hi,

First of all, thank you for your work, your extension works fine on my store.
 
I'm using the Woocommerce android app to check-up my orders, and unfortunetly the selected pick-up time does not appear.
Is that a way for you to make my dream come true?

Thanks,

**Reply by timnolte** [reply]
I'm not sure if this will be possible, as often the mobile apps are restricted to just core functionality. I will have to take a look and see what options may exist.

**Reply by tnchuntic** [reply]
@timnolte I think you can do it by adding the date time slot selected to the notes. We have done it on the Icon Delivery Timeslot plugin... client was using Woocommmerce app to check orders.

**Comment by An0dmin** [comment]
Ok thanks
